core/templates/core/campaign.html

```diff
@@ -121,41 +120,15 @@ <h2 id="related-content-heading" class="heading-large margin-top-60">{{ page.rel
     </div>
 
     <ul class="grid-row" aria-labelledby="related-content-heading" id="related-pages-list">
-      {% if page.related_page_one_heading and page.related_page_one_url %}
-        <li class="column-third" id="related-page-one">
-          {% if page.related_page_one_image %}
-            <img id="related-page-one-image" src="{{ page.related_page_one_image.url }}" alt="">
-          {% endif %}
-          <a id="related-page-one-link" href="{{ page.related_page_one_url }}" class="heading-medium grey-text margin-top-30">{{ page.related_page_one_heading }}</a>
-          {% if page.related_page_one_description %}
-            <p id="related-page-one-description">{{ page.related_page_one_description }}</p>
-          {% endif %}
-        </li>
-      {% endif %}
-
-      {% if page.related_page_two_heading and page.related_page_two_url %}
-        <li class="column-third" id="related-page-two">
-          {% if page.related_page_two_image %}
-            <img id="related-page-two-image" src="{{ page.related_page_two_image.url }}" alt="">
-          {% endif %}
-          <a id="related-page-two-link" href="{{ page.related_page_two_url }}" class="heading-medium grey-text margin-top-30">{{ page.related_page_two_heading }}</a>
-          {% if page.related_page_two_description %}
-            <p id="related-page-two-description">{{ page.related_page_two_description }}</p>
+      {% for article in page.related_pages %}
+        <li class="column-third" id="related-page-{{ article.meta.slug }}">
+          {% if article.article_image_thumbnail %}
+            <img id="related-article-{{ article.meta.slug }}-image" src="{{ article.article_image_thumbnail.url }}" alt="">
           {% endif %}
+          <a id="related-page-{{ article.meta.slug }}-heading" href="/prototype{{ article.full_path }}" class="heading-medium grey-text margin-top-30">{{ article.article_title }}</a>
```

Comment:
looks like non-prototype pages are linking to prototype pages: this template is used by `CampaignPageView`.

is that ok?

Reply:
yeah for now. it's a separate PR to remove all the prototype prefixing